resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/repository/JpaPersistenceLayer.java

```diff
@@ -101,8 +108,12 @@ public JpaPersistenceLayer(final CompletenessRepository completenessRepository,
      */
     private boolean isResourceIdComplete(final String resourceId) {
         // Check details with completeness db
-        boolean complete = completenessRepository.compositeExistsByEntityTypeAndEntityId(EntityType.RESOURCE, resourceId);
-        LOGGER.debug("Resource {} is {}", resourceId, complete ? "complete" : "not complete");
+        boolean complete = completenessRepository.existsByEntityTypeAndEntityId(EntityType.RESOURCE, resourceId);
+        if (complete) {
+            LOGGER.debug(RESOURCE_IS, resourceId, COMPLETE);
+        } else {
+            LOGGER.debug(RESOURCE_IS, resourceId, NOT_COMPLETE);
+        }
```

Comment:
Was this change from a previous comment. It's fine but I much preferred the compactness and readability of the old code.

Reply:
Yeah, in this case i think the old method looked better, but we've set sonarqube to reject any use of ternary operators...